modules/paginator/main/index.coffee

```diff
@@ -1,133 +1,320 @@
+hx.userFacingText({
+  paginator: {
+    paginatorAria: 'Pagination navigation',
+    currentPageAria: 'Current page, page $page',
+    gotoPageAria: 'Goto page $page',
+    prevPageAria: 'Goto previous page, page $page',
+    nextPageAria: 'Goto next page, page $page',
+    prev: 'Prev',
+    next: 'Next',
```

Comment:
How would we pass in different languages?

Reply:
Two methods, one is to pass to the constructor (which is where these values are consumed) on a per-component basis or you can override `hx.userFacingText`:
```
new Paginator('#selector', {
  prevText: 'Bob!'
})
```

```
hx.userFacingText({
  paginator: { prev: 'Bob!' }
})

new Paginator('#selector') // Uses `Bob!`
```